src/3rd_party/CMakeLists.txt

```diff
@@ -30,7 +30,10 @@
 
 set(3RD_PARTY_INSTALL_PREFIX $ENV{THIRD_PARTY_INSTALL_PREFIX})
 if(NOT 3RD_PARTY_INSTALL_PREFIX)
-  set(3RD_PARTY_INSTALL_PREFIX ${CMAKE_BINARY_DIR}/bin)
+  set(3RD_PARTY_INSTALL_PREFIX /usr/local)
```

Comment:
@anosach-luxoft  please add task to refactor logger system. and add TODO there
Do not install logger in /usr/local/lib by default